### PR TITLE
Arkan - Ergoscript course HW1

### PR DESCRIPTION
**Pull request by arkan294**

### CODE REVIEW

File: ErgoScript-Developer-Course/batch-0/Arkan/HW1_Multi-Stage_Transaction.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+```
+{
+	val NFTprice: Long = getVar[Long](0).get
+	val sellerPK: SigmaProp = getVar[SigmaProp](1).get
+
+	val NFTsold: SigmaProp = OUTPUTS.exists { (out: Box) =>
+		out.value >= NFTprice &&
+		out.propositionBytes == sellerPK.propBytes
+	}
+	NFTsold || sellerPK
```
Comment (k3ithl1m) [inline ErgoScript-Developer-Course/batch-0/Arkan/HW1_Multi-Stage_Transaction.md:123]: sigmaProp(NFTsold || sellerPK)

**Comment by kii-dot** [comment]
I'm curious to know if you tested this script?

**Comment by arkan294** [comment]
I haven't tested it, just checked if it compiles. I thought about testing when I learn more about the appkit.

**Comment by arkan294** [comment]
@k3ithl1m I've tested it (and even chained the transactions so that everything happens in one block) and it seems to be working, but I had to reference the NFT box's script (the one with atLeast(...)) in the Pooling boxes using Coll[Byte] of already compiled script instead of putting it in a val.

**Pull request merged**